smartsim/_core/utils/serialize.py

```diff
@@ -238,6 +234,18 @@ def _dictify_db(
                 "conf_file": shard.cluster_conf_file,
                 "out_file": out_file,
                 "err_file": err_file,
+                "memory_file": (
+                    str(status_dir / "memory.csv") if db.telemetry.is_enabled else ""
+                ),
+                "client_file": (
+                    str(status_dir / "client.csv") if db.telemetry.is_enabled else ""
+                ),
+                "client_count_file": (
+                    str(status_dir / "client_count.csv")
+                    if db.telemetry.is_enabled
+                    else ""
+                ),
+                "collectors": "1" if db.telemetry.is_enabled else "0",
```

Comment:
I chose the stringified form of a numeric because it was consistent with `SMARTSIM_FLAG_TELEMETRY` and casting "False" to boolean gives me True, but casting 0 or "0" via `bool(int(value))` doesn't have the "all non-empty strings are True" cast gotcha

Reply:
Gotcha!! That makes sense to me!!